strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/AuthenticatorsRegistrySerializer.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.authentication.registry.support;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+import org.carlspring.strongbox.authentication.registry.AuthenticatorsRegistry;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class AuthenticatorsRegistrySerializer
```

Comment:
@fuss86 you can dynamically add classes to `org.springframework.oxm.jaxb.Jaxb2Marshaller`, if I understand the problem correctly.